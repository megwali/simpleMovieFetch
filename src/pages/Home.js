const Home = () => (
  <div>
    <h2>Homie</h2>
  </div>
);

export default Home;
