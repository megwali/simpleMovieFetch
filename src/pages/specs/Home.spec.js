import { create } from 'react-test-renderer';
import Home from '../Home';

describe('Home', () => {
  it('renders', () => {
    const component = create(<Home />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
