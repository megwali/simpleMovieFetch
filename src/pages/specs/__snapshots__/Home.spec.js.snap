// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders 1`] = `
<div>
  <h2>
    Homie
  </h2>
</div>
`;
